3.analyze_data/compare_raw_mitocheckjump.ipynb

```diff
@@ -0,0 +1,696 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #39.            results = stats.kstest(mitocheck_distribution, jump_distribution)
</pre><p>The defaults should work here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/JUMP-single-cell/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Thanks for checking!